animation.h

```diff
@@ -3,13 +3,24 @@
 
 #include "location.h"
 
-enum class AnimationType { kDefault, kRunRight, kRunLeft };
+enum class AnimationType {
+  kDefault,
+  kRunRight,
+  kRunLeft,
+  kSSTransformation,
```

Comment:
Even for acronyms (SS for super saiyan), use CamelCase

kSsTransformation

https://google.github.io/styleguide/cppguide.html#Constant_Names